src/garage/torch/algos/ppo.py

```diff
@@ -49,17 +51,19 @@ def __init__(self,
                  baseline,
                  optimizer=torch.optim.Adam,
                  policy_lr=_Default(3e-4),
-                 max_path_length=500,
-                 lr_clip_range=2e-1,
-                 num_train_per_epoch=1,
-                 discount=0.99,
-                 gae_lambda=0.97,
-                 center_adv=True,
-                 positive_adv=False,
-                 policy_ent_coeff=0.0,
-                 use_softplus_entropy=False,
-                 stop_entropy_gradient=False,
-                 entropy_method='no_entropy'):
+                 max_path_length=_Default(500),
```

Comment:
Is there any particular reason to use `_Default` everywhere here? It's somewhat weird. Usually wrappers are only used on default objects for unusual cases (in particular, where `None` must be a valid argument, or detecting that an argument was passed is particularly important).

Reply:
Then I need to put `_Default` on minibatch_size for VPG (default value is None) except every other parameters?
How about `max_optimization_epochs` parameter?

Reply:
is this resolved?

Reply:
Please only put `_Default` on parameters that need it, and write a comment why.